File: Geeks-For-Geeks/000-Mathematics/002.palindrome.py
# Time Complexity: O(d)
def reverse(n):
    rev = 0
    temp = n
    while temp:
        rev = rev * 10 + temp % 10
        temp = temp // 10
    return rev

def isPalindrome(n):
    return n == reverse(n)

print(isPalindrome(8008))

File: Geeks-For-Geeks/000-Mathematics/002.palindrome.js
const isPalindrome = function (n) {
  const num = (n + '').split('');
  return num.join('') === num.reverse().join('');
};

const isPalindrome2 = (n) => {
  return n === parseInt((n + '').split('').reverse().join(''));
};

console.log(isPalindrome2(909));
